app/templates/session-expired.html

```diff
@@ -5,12 +5,17 @@
 
 {% block main %}
 
-  <div class="u-mb-s">
+<div class="u-mb-s">
     <h1 class="saturn" {{helpers.track('visible', 'Timeout', 'Timeout expired')}}>Your session has expired</h1>
-  </div>
+</div>
 
-  <div class="panel panel--simple panel--success panel--spacious">
+  <div class="panel panel--simple panel--success panel--spacious u-mb-s">
     <p>To help protect your information we have signed you out.</p>
   </div>
 
+{% if env == EQ_PREPROD_ENV %}
```

Comment:
How does the app know what environment it is in ?

Reply:
The URL should be pulled from the variable. not using the variable to select a hard coded URL